binding.gyp

```diff
@@ -15,7 +15,8 @@
           './src/'
       ],
       'libraries': [
-        '<!@(pkg-config libosrm --libs)'
+        '<!@(pkg-config libosrm --libs)',
+        '-ltbb'
```

Comment:
The proper way of handling this would be to fix it in OSRM's CMake script that generated the pkg-config file. (But that's a valid workaround for now I guess..)